### PR TITLE
ISPN-12300 Unit testing first steps

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-12300

### CODE REVIEW

**Comment by karesti** [comment]
@pruivo whenever you can

**Comment by karesti** [comment]
@pruivo please grab a look again and merge if it works as expected

File: data/create-data.sh [hunk 1]
```diff
@@ -3,17 +3,17 @@ pass=$2
...
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/programmers
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/seniors
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/people
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/example
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/local_users
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/invalidated
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/invalidated-sync
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/xsiteCache
-curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/xsite-transactional
+#curl -XDELETE  -u $userPass http://localhost:11222/rest/v2/caches/users
```
Comment (pruivo) [inline data/create-data.sh:6]: is this intentional?

Comment (karesti) [reply data/create-data.sh]: partially. I comited a change

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**